doc/configuring.rst

```diff
@@ -180,187 +180,6 @@ The example configuration file declares the following `Universum` steps:
 4. Copy resulting directory `"out"` to the artifact directory
 
 
-.. _keys:
```

Comment:
Please make sure there's no broken links to 'keys' left in this document. And in change log.